integration_tests/tests/forseti/controls/scanner-location.rb

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+require 'securerandom'
+
+db_password = attribute('forseti-cloudsql-password')
+db_user_name = attribute('forseti-cloudsql-user')
+forseti_server_storage_bucket = attribute('forseti-server-storage-bucket')
+model_name = SecureRandom.uuid.gsub!('-', '')[0..10]
+
+control 'scanner-location' do
```

Comment:
(minor) Can you rename this file/control to be `scanner-location_scanner` to align with the naming convention for the other scanner tests?